src/it/scala/uk/gov/hmrc/hmrcfrontend/support/TemplateIntegrationSpec.scala

```diff
@@ -39,6 +45,8 @@ abstract class TemplateIntegrationSpec[T: OWrites: Arbitrary](hmrcComponentName:
   override def overrideParameters(p: Test.Parameters): Test.Parameters =
     p.withMinSuccessfulTests(20)
 
+  val component = app.injector.instanceOf[C]
```

Comment:
@natcarey with this refactoring could we now move the render method to the abstract class too? Seems like it's now basically the same code except for the type of the render param but we have that in T

Reply:
Ooh, that would be lovely.  I'll try that. :)